.circleci/get_pinned_chrome_version.sh

```diff
@@ -46,7 +47,18 @@ echo "$(GREEN "Chrome version history URL is") $(CYAN "${CHROME_VERSION_HISTORY_
 # Determine the Chrome version
 # See https://developer.chrome.com/docs/versionhistory/guide
 echo "$(GREEN "Determining Chrome version...")"
-CHROME_VERSION="$(curl -sS ${CHROME_VERSION_HISTORY_URL} | jq -r ".versions[]|.version" | grep -m 1 "${CHROME_MAJOR_VERSION}\.")"
+for attempt in {2..0}
+do
+  CHROME_VERSION_HISTORY_JSON="$(curl -sS ${CHROME_VERSION_HISTORY_URL})"
+  if [[ $(echo ${CHROME_VERSION_HISTORY_JSON} | jq -r .error) == "null" ]]; then
```

Comment:
is `"null"` the only error that can be returned? I'm concerned that hard-coding an expected value could let other kinds of errors slip through.

Reply:
`null` just means that the JSON object _didn't_ have an `"error"` field. Punted due to the other comment